js_modules/dagit/src/schema.graphql

```diff
@@ -273,7 +278,7 @@ type MissingFieldConfigError implements PipelineConfigValidationError {
 }
 
 type Mutation {
-  startPipelineExecution(config: PipelineConfig, pipeline: ExecutionSelector!): StartPipelineExecutionResult!
+  startPipelineExecution(config: PipelineConfig, executionMetadata: ExecutionMetadata, pipeline: ExecutionSelector!, reexecutionConfig: ReexecutionConfig, stepKeys: [String!]): StartPipelineExecutionResult!
```

Comment:
do we have to worry about backwards compatibility much? Does seem having a separate call for `reexecutePipeline` is probably where we want to end up